lib/core/test.js

```diff
@@ -11,9 +11,9 @@ var Config = require('./config.js');
 var RunCode = require('./runCode.js');
 var TestLogger = require('./test_logger.js');
 
-var Test = function(_options) {
-  var options = _options || {};
-  var simOptions = options.simulatorOptions || {};
+var Test = function(options) {
```

Comment:
It's usually convention to not have underscores before names of javascript variables unless they're private/unused, etc.

Also, variable reuse is fine, meaning the old version would have been fine as:

```
var Test = function(options) {
  options = options || {};
  ...
```